app/controllers/events_controller.rb

```diff
@@ -81,6 +81,12 @@ def set_user
 
     # Only allow a trusted parameter "white list" through.
     def event_params
+      if params.has_key? :league
+        params[:event] = params.delete :league
+      elsif params.has_key? :tournament
+        params[:event] = params.delete :tournament
+      end
```

Comment:
All these lines are only nessecarry aslong as we don't actually create League or Tournament Objects. I think we should change that either before merging this or in one of the next prs, so that other teams also know how to properly use the model.